week-1/fix/median.js

```diff
@@ -3,9 +3,25 @@
 // If you're in the week-1 directory, you can run npm test -- fix to run the tests in the fix directory
 
 function calculateMedian(list) {
-  const middleIndex = Math.floor(list.length / 2);
-  const median = list.splice(middleIndex, 1)[0];
-  return median;
+  const sortedList = list.sort((a, b) => a - b);
+  const middleIndex = Math.floor(sortedList.length / 2); 
+  const median = sortedList.slice(middleIndex-1);    // 1 2 3 4 = 3(because 1 is 0 index)/2=1.5 (math.floor will give the integer which is 1, the index 1 is 2)
+  if (list.length % 2 === 0){
+    return (sortedList[middleIndex] + sortedList[middleIndex-1])/2;
+  }
+  return sortedList[middleIndex]; 
 }
 
-module.exports = calculateMedian;
+module.exports = calculateMedian; // this is to link this file with the test file 
+
+// to run tests on this go int he specific folder and file by cd and then write npm test median 
+
```

Comment:
Nicely done, but can you please remove all the commented codes and comments from this files please unless is a useful comment that describes functionality other than that is not the best practice to push commented code or useless comments.
Also, you can add a test file to test your functionality. 
Always make sure to clean your code before you push!